### ISSUE TITLE
Do you have the code of Pytorch version?

### ISSUE THREAD

**Issue by ma3252788** [YimianDai/open-aff#2]
Or will you consider publishing the pytorch version of the code in the future?

**Comment by YimianDai** [comment]
I am going to convert to PyTorch since it is the most dominant framework. It is a bit meaningless to release the code in a framework that few people are using. 

But not in the near future. I am in quite a chaos and uncertain status recently. 

If everything is set, I am planning to re-implement it in PyTorch in the coming spring festival.

**Comment by ma3252788** [comment]
> I am going to convert to PyTorch since it is the most dominant framework. It is a bit meaningless to release the code in a framework that few people are using, although I like MXNet very much.
> 
> But not in the near future. I am in quite a chaos and uncertain status recently.
> 
> If everything is set, I am planning to re-implement it in PyTorch in the coming spring festival.

Okay, thank you very much, I look forward to seeing the pytorch version of aff in the near future~

**Comment by majunfu** [comment]
Waiting for your pytorch version

**Comment by Senwang98** [comment]
@YimianDai  Waiting for your pytorch version

**Comment by YimianDai** [comment]
@1187697147 Hi, @bobo0810 has contributed a Pytorch version https://github.com/YimianDai/open-aff/tree/master/aff_pytorch. Although it is not written by me, I hope it will help you. 

As to me, currently, I am busy with finalizing my PhD Oral defense. I think I will have a lot of free time next week (I hope so) and will start to do all the things that I have ever promised in my replied issues of this repo. 

Sorry for the delay.

**Comment by Senwang98** [comment]
@YimianDai  Nice, good luck to you and thanks for your work!

**Comment by bobo0810** [comment]
Description:
1. MS_CAM, AFF, iAFF fusion operation are completely consistent with the MXNet version.
2. AFF-ResNet is  Pytorch official ResNet + AFF/iAFF.

Good luck!:smiley:

**Comment by hiyyg** [comment]
@bobo0810 @YimianDai Thanks for your great work, do you have pretrained models for pytorch version?

**Comment by bobo0810** [comment]
> @bobo0810 @YimianDai Thanks for your great work, do you have pretrained models for pytorch version?

Unfortunately:pensive:, I may not be able to provide it. Because I only apply it to the company's private data set.

**Comment by YimianDai** [comment]
@hiyyg Hi, many thanks for being interested in my work. I am considering to fully convert to PyTorch and extend the conference paper to a journal article with more experiments. 

If the accuracy of my new PyTorch implementation (not done yet) does not drop, I will release the pretrained models for PyTorch version. Please stay patient. I will update the process in this issue in the following days.

**Comment by ma3252788** [comment]
> @hiyyg Hi, many thanks for being interested in my work. I am considering to fully convert to PyTorch and extend the conference paper to a journal article with more experiments.
> 
> If the accuracy of my new PyTorch implementation (not done yet) does not drop, I will release the pretrained models for PyTorch version. Please stay patient. I will update the process in this issue in the following days.

What's the result, please?

**Comment by YimianDai** [comment]
I am working on this. The next version will be PyTorch and Open-MMLab based.

**Comment by ma3252788** [comment]
> I am working on this. The next version will be PyTorch and Open-MMLab based.

Got it. Thank you for your reply. Looking forward to your work

**Comment by fhaji75** [comment]
> @hiyyg Hi, many thanks for being interested in my work. I am considering to fully convert to PyTorch and extend the conference paper to a journal article with more experiments.
> 
> If the accuracy of my new PyTorch implementation (not done yet) does not drop, I will release the pretrained models for PyTorch version. Please stay patient. I will update the process in this issue in the following days.

Hello, thanks for your good repository, Can you release pre-trained models for Pytorch?

**Comment by CaoZhonglei** [comment]
请问有tensorflow版本的代码吗？
